inc/settings/namespace.php

```diff
@@ -71,14 +71,13 @@ function render_settings_page() {
 		<h1><?php esc_html_e( 'FAIR Settings', 'fair' ); ?></h1>
 		<p><?php printf(
 			/* translators: %s: AspirePress website URL */
-			esc_html__( 'Your site is now disconnected from WordPress.org managed servers and uses a dedicated mirror from %s. In the coming weeks and months, we\'ll be making the mirror configurable and add in the ability to add your own.', 'fair' ),
-			'<a href="https://aspirepress.org/">AspirePress</a>'
+			esc_html__( 'Your site is now disconnected from WordPress.org managed servers and uses a dedicated mirror from %1$sAspirePress%2$s. In the coming weeks and months, we\'ll be making the mirror configurable and add in the ability to add your own.', 'fair' ),
+			'<a href="https://aspirepress.org/" target="_blank">', '</a>'
```

Comment:
I don't think including `target="_blank"` should be mixed in with this string change. Opening links in a new tab is somewhat of a hot topic, so I think it's best discussed/handled elsewhere.

Reply:
Agreed. I also don't think that this specific text should be translatable; it's a proper noun, which shouldn't generally be translated.